docs/sources/get-started/overview.ja.md

```diff
@@ -0,0 +1,47 @@
+---
+menuTitle: Lokiの概要
+title: Lokiの概要
+description: Lokiの製品概要と機能
+---
+# Lokiの概要
+
+Lokiは、[Prometheus](https://prometheus.io/)にインスパイアされた水平スケーラブルで高可用性のマルチテナントログアグリゲーションシステムです。Lokiは、メトリクスではなくログに焦点を当て、プルではなくプッシュでログを収集する点でPrometheusとは異なります。
+
+Lokiは非常にコスト効率が高く、高いスケーラビリティを持つように設計されています。他のログシステムとは異なり、Lokiはログの内容をインデックス化せず、各ログストリームのラベルセットとしてログのメタデータのみをインデックス化します。
+
+ログストリームとは、同じラベルを共有する一連のログのことです。ラベルは、Lokiがデータストア内でログストリームを見つけるのに役立つため、質の高いラベルセットを持つことが効率的なクエリ実行の鍵となります。
+
+ログデータは圧縮され、Amazon Simple Storage Service (S3) や Google Cloud Storage (GCS) などのオブジェクトストア、または開発やコンセプト実証のためにファイルシステムにチャンクとして保存されます。小さなインデックスと高度に圧縮されたチャンクにより、操作が簡素化され、Lokiのコストが大幅に削減されます。
+
+{{< figure  src="/media/docs/loki/loki-overview-2.png" caption="**Lokiログスタック**" >}}
+
+典型的なLokiベースのログスタックは、3つのコンポーネントで構成されています：
+
+- **エージェント** - エージェントまたはクライアント、例えばLokiに同梱されているPromtailやGrafana Agent。エージェントはログを収集し、ラベルを追加してログをストリームに変換し、HTTP APIを通じてLokiにストリームを送信します。
```

Comment:
Google Translate says that this line reads "Agent - An agent or client, such as Promtail or Grafana Agent that ship with Loki."  Please change this to reference Alloy, as the Loki documentation has replaced references to Grafana Agent with Grafana Alloy.